package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "polkadot-deployer",
-  "version": "1.3.2",
+  "version": "1.3.3",
```

Comment:
or 1.4.0 ?

Reply:
could this be seen differently because of the "substrate-telemetry": "v2.2.0" upgrade ?

Reply:
i think we are not breaking backwards compatibility, but we are introducing new features, as i see it we should bump minor, wdyt?